libs/navigation/website/src/lib/navbar/navbar.tsx

```diff
@@ -145,106 +155,155 @@ export const NavPaperWrapper = styled('div')`
   }
 `
 
 export const NavPaperCategoryLinks = styled('div')`
   display: grid;
-  gap: ${({theme}) => theme.spacing(2)};
-  text-transform: uppercase;
   font-weight: ${({theme}) => theme.typography.fontWeightMedium};
   font-size: ${({theme}) => theme.typography.h6.fontSize};
 `
 
 export const NavPaperMainLinks = styled(NavPaperCategoryLinks)`
+  gap: ${({theme}) => theme.spacing(1)};
+`
+
+export const NavPaperChildrenWrapper = styled('div')`
+  position: relative;
+  padding: ${({theme}) => theme.spacing(1.5)};
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(40px, 1fr));
+  justify-items: center;
+  left: 0;
+  top: 0;
+  width: 100%;
+
   ${({theme}) => css`
-    ${theme.breakpoints.up('sm')} {
-      display: none;
+    ${theme.breakpoints.up('md')} {
+      position: absolute;
+      grid-template-columns: auto;
+      justify-items: start;
+      width: calc(100% / 6);
+      gap: ${theme.spacing(3)};
+      padding-top: ${theme.spacing(10)};
+      padding-left: ${theme.spacing(4.5)};
     }
   `}
 `
 
 const NavPaper = ({
   main,
   categories,
-  closeMenu
+  closeMenu,
+  children
 }: {
   main: FullNavigationFragment | null | undefined
-  categories: FullNavigationFragment[]
+  categories: FullNavigationFragment[][]
   closeMenu: () => void
+  children: React.ReactNode
 }) => {
   const {
-    elements: {Link, H5}
+    elements: {Link, H4, H6}
   } = useWebsiteBuilder()
   const theme = useTheme()
 
   return (
     <NavPaperWrapper>
+      {children && <NavPaperChildrenWrapper>{children}</NavPaperChildrenWrapper>}
       {!!main?.links.length && (
         <NavPaperMainLinks>
           {main.links.map((link, index) => {
             const url = navigationLinkToUrl(link)
 
             return (
               <Link href={url} key={index} color="inherit" underline="none" onClick={closeMenu}>
-                {link.label}
+                <H4>{link.label}</H4>
               </Link>
             )
           })}
         </NavPaperMainLinks>
       )}
-
       {!!categories.length && (
-        <NavPaperCategoryList>
-          {categories.map(nav => (
-            <NavPaperCategory key={nav.id}>
-              <H5 component="div">{nav.name}</H5>
-
-              <NavPaperCategoryLinks>
-                {nav.links?.map((link, index) => {
-                  const url = navigationLinkToUrl(link)
-
-                  return (
-                    <Link
-                      href={url}
-                      key={index}
-                      color="inherit"
-                      underline="none"
-                      css={navPaperLinkStyling(theme)}
-                      onClick={closeMenu}>
-                      {link.label}
-                    </Link>
-                  )
-                })}
-              </NavPaperCategoryLinks>
-            </NavPaperCategory>
+        <>
+          {categories.map((categoryArray, arrayIndex) => (
+            <LinksGroup key={`category-group-${arrayIndex}`}>
```

Comment:
```suggestion
            <LinksGroup key={arrayIndex}>
```

This key doesn't add anything